src/main/java/org/jenkinsci/confluence/plugins/JenkinsPluginInfoMacro.java

```diff
@@ -157,6 +157,10 @@ public String execute(Map parameters, String body, RenderContext renderContext)
                                         .append("]\n[Since Latest Release|").append(fisheyeBaseUrl)
                                         .append(releaseTimestamp).append(fisheyeEndUrl).append(']');
                         }
+                        if (isGithub) {
+                            String buildhiveUrl = "https://buildhive.cloudbees.com/job/jenkinsci/job/" + sourceDir;
+                            toBeRendered.append("[!").append(buildhiveUrl).append("/badge/icon|border=0!|").append(buildhiveUrl).append("/]");
```

Comment:
Please remove this badge icon from macros, any failed PR shows failed icon and i see no way to disable it.